packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -0,0 +1,112 @@
+import React              from 'react';
+import PropTypes          from 'prop-types';
+import {
+    Button,
+    Dialog,
+    Loading }         from 'deriv-components';
+// import { WS } from '../../utils/websocket';
+import                         './buy-sell.scss';
```

Comment:
this should be on the last line

Reply:
of the import section, I mean

Reply:
im afraid popup.jsx wont be able to access the scss if i put the scss at the bottom